### PR TITLE
perf(GraphQL): RHICOMPL-2928 preload last scanned for hosts

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
b50ac05e-fd34-4146-946c-86b3750ec0c5
db5d18c0-fd19-4aee-bae5-d36d7301bff5

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
2782cd9d-240c-4e15-aca4-2eb12c5fff96
b2bec320-a034-457b-8790-effe4dae55e1

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
029c544d-d70c-4b5c-a1b8-27e9cc936073
3ce05009-20d4-4ad0-8b68-cd2b5506a34c

File: test/graphql/queries/system_query_test.rb [hunk 1]
```diff
@@ -75,12 +78,30 @@ class SystemQueryTest < ActiveSupport::TestCase
...
     )
 
-    assert_equal 4, result['data']['system'].count
+    assert_equal 5, result['data']['system'].count
     result['data']['system'].each do |_, timestamp|
       assert_equal timestamp, Time.parse(timestamp).iso8601
     end
   end
 
+  test 'query host lastScanned returns never if no test results' do
```
Comment (romanblanco) [inline test/graphql/queries/system_query_test.rb:87]:
```suggestion
  test 'query host lastScanned returns "Never" if no test results' do
```

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
3795ba88-104d-4522-93f7-d816d35554ea
b66f00b0-2e74-4e7a-a760-3bb9ed123905

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
dab709f4-70b0-473f-bacf-5a16479a735b
de94fc23-c843-45d7-a82a-56c9765789f6

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
1408afc8-3609-4488-9a01-a5044f5dfd4d
4086da54-4bac-4db0-a5fa-06bcd64c6423

**Pull request merged**